### ISSUE TITLE
Mopidy crashes when using MPRIS commands

### ISSUE THREAD

**Issue by Fubukimaru** [tkem/mopidy-dleyna#73]
Hello,

First thanks for your work on this piece of software :).

After installing mopidy-dleyna and dleyna-server, mopidy is able to find the content of the DLNA server (minidlna) and it is also able to play the music. However after a short period of time, mopidy keeps on playing the  music but the mopidy server is not accessible to clients. And after the song is over, it won't reproduce the next one, it will remain stuck.

Mopidy and dleyna-server won't give any error message in the logs. Is this a known issue? Is there any way I can test things and get information on what is failing?

Thanks!

**Comment by tkem** [comment]
No, this is not a known issue. I've been using mindlna myself from the start, and I've found it to be the most reliable and feature complete DLNA server I've encountered so far. So sorry, no clue.

Log files might help, especially when running mopidy with `-v` - hey, I just found out there's now a config option for that: https://docs.mopidy.com/en/latest/config/#logging-section ;-)

**Comment by Fubukimaru** [comment]
Hello!

Thanks for the information, I have found what triggers the issue! I use i3 with i3blocks as informational bar and it has a plugin that uses `playerctl` to get the song information.

In this case with `playerctl metadata` it should show you the song metadata, but with songs from the DLNA server it gets stuck (and mopidy also gets stuck).

The log gives the following:
```
Mar 04 21:34:15 Yae python3[70179]: DEBUG    2021-03-04 21:34:15,754 [70179:dLeynaBackend-8] mopidy_dleyna.util
Mar 04 21:34:15 Yae python3[70179]:   Calling D-Bus method SearchObjectsEx('Path = "/com/intel/dLeynaServer/server/uuid_3a4d696e69_2d444c_2d164e_2d9d41_2d00e04c36042e/363424302432462430"', dbus.UInt32(0), dbus.UInt32(0), ['AlbumArtURL', 'Path', 'RefPath'], '')
```

A regular mp3 file prints the following:
```
playerctl metadata
mopidy mpris:trackid             '/com/mopidy/track/1'
mopidy mpris:length              1183764000
mopidy xesam:url                 local:track:TheFlowerKings/The%20Flower%20Kings%20-%20Adam%20%26%20Eve/01%20-%20Love%20Supreme.mp3
mopidy xesam:title               Love Supreme
mopidy xesam:artist              The Flower Kings
mopidy xesam:album               Adam & Eve
mopidy xesam:albumArtist         The Flower Kings
mopidy mpris:artUrl              /local/ea465b7139c09e9c077654f6fb382de9-300x300.jpeg
mopidy xesam:trackNumber         1
```

So I assume that there is something missing in the `mopidy mpris:artUrl` part.

Any clue on how to fix this?

Thanks!

Edit:
It also happens with other commands like `playerctl next`.
